src/Microsoft.AspNetCore.Blazor.Browser.JS/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "blazor.host.client",
-  "version": "0.0.1",
```

Comment:
We don't ever need to change this, as we're not shipping it as an NPM package. The version number isn't used anywhere, but there does have to be some semver-legal number there otherwise NPM tools complain.

Reply:
To avoid future confusion about the need for updates it would probably be better to revert `package.json` change. I only picked `0.0.1` rather than `1.0.0` to avoid the implication that it was a finished product.

Reply:
OK good to know. I will undo this part of the change. If only we could put a comment here.....